src/main/java/io/candydoc/ddd/aggregate/AggregateFound.java

```diff
@@ -14,7 +14,7 @@ public class AggregateFound implements Event {
   @NonNull String description;
   @NonNull String canonicalName;
   @NonNull String packageName;
-  @NonNull String boundedContext;
+  @NonNull String domainContext;
```

Comment:
What is a "domainContext" ? 

I'm not sure that this concept comes from DDD. It might be either a domain or a boundedContext.

Reply:
You can read this : https://medium.com/nick-tune-tech-strategy-blog/domains-subdomain-problem-solution-space-in-ddd-clearly-defined-e0b49c7b586c 
I heard about domain in "DDD Distilled" book from Vaugh Vernon.